Automattic/jetpack
# Social: Create the Jetpack Social version of the Publicize panel

#### Changes proposed in this Pull Request:

Jetpack Social needs its own implementation of the Publicize panel,
displayed in the editor sidebar. This is because we're not handling paid
plans yet, and the paid features (Republicize) aren't available. When we
do, it's unlikely that we'll handle everything in the same way as the
Jetpack plugin.





The dependent PRs - #24464, #24408 and #24324 - are merged, so the panel is now fully operational!

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Build the Jetpack Social plugin
- Activate it and connect
- Open the block editor and observe that you have the Publicize sharing panel in the sidebar.
- You can enable and disable connections, and Publicize the post on publish, with an optional custom message

<img width="1225" alt="image" src="https://user-images.githubusercontent.com/96462/170381070-9160c064-873d-4544-9914-b82e6c8e8a7c.png">
